app/services/widgets/settings/event-list.ts

```diff
@@ -134,11 +134,7 @@ export class EventListService extends WidgetSettingsService<IEventListData> {
           ),
         },
       ],
-      mixer: [
-        { key: 'sparks_minimum', title: $t('Min. Sparks') },
-        { key: 'embers_minimum', title: $t('Min. Embers') },
-      ],
-    }[platform];
+    }[platform as 'twitch'];
```

Comment:
why are we even making this keyed object if there's only 1 key lol

Reply:
Because this object accepts reading only for the  `twitch` key, meanwhile `TPlatfom` type can be any of platforms.

Reply:
yea i guess i'm asking why this is a keyed object in the first place if it only accepts one key

Reply:
We had Mixer in that list before. I strived to not touch logic in this PR if I can to avoid occasional issues. So I just added a typecast here